src/noisepy/seis/datatypes.py

```diff
@@ -239,6 +242,24 @@ def get_storage_options(self, path: str) -> Dict[str, Any]:
     def dt(self) -> float:
         return 1.0 / self.samp_freq
 
+    def load_stations(self, stations_list=None) -> List[str]:
+        if stations_list is None:
+            # Load the list from the file
+            with open(self.stations_file, "r") as file:
+                self.stations = file.read().splitlines()
+        else:
+            self.stations = stations_list
+        return 0
+
+    def save_stations(self, value: List[str]):
```

Comment:
I think this should save `self.stations` instead of taking a `value` parameter. But we can defer this change for the next PR that makes all of this transparent.